doc-content/jbpm-docs/src/main/asciidoc/ElasticSearch/integration-elasticsearch-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='integration-elasticsearch-proc_{context}']
+= Configuring the {KIE_SERVER} to send information to ElasticSearch when a transaction is committed
+
+You can configure the {KIE_SERVER} to send information to ElasticSearch about every event when a process, case, or task is completed. The {KIE_SERVER} sends the information when it commits transactions.
```

Comment:
Change "completed" to "comitted". "Completed" is a legit state for process, cases and task and events are not only published when this state is reached, but when the process, case or task is modified in any way